src/_includes/project.html

```diff
@@ -1,24 +1,36 @@
-
 <h3>
-    <a href="{{ project.link }}">{{ project.name }}</a>
+  {% if project.link %}
+  <a target="_blank" href="{{ project.link }}">{{ project.name }}</a>
+  {% elsif project.link == nil %}
+  {{ project.name }}
```

Comment:
Project headings that aren't links are not getting the same left-padding as ones that are links
![Screen Shot 2019-07-30 at 7 59 43 PM](https://user-images.githubusercontent.com/20404311/62180498-1a5cab80-b305-11e9-9bd1-e5dac2598837.png)

Reply:
![Screen Shot 2019-07-30 at 8 00 09 PM](https://user-images.githubusercontent.com/20404311/62180610-82ab8d00-b305-11e9-9094-25a9627fd608.png)

Reply:
Boo.